<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectFocHall</class>
 <widget class="QWidget" name="DetectFocHall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Detect Hall Sensors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="helpButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Show help</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/Help-96.png</normaloff>:/res/icons/Help-96.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="currentBox">
          <property name="toolTip">
           <string>Open loop current to turn the motor with</string>
          </property>
          <property name="prefix">
           <string>I: </string>
          </property>
          <property name="suffix">
           <string> A</string>
          </property>
          <property name="maximum">
           <double>300.000000000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Start detection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/Circled Play-96.png</normaloff>:/res/icons/Circled Play-96.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Apply detection result</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/apply.png</normaloff>:/res/icons/apply.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>45</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QSpinBox" name="hall0Box">
          <property name="toolTip">
           <string>Hall table 0 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hall1Box">
          <property name="toolTip">
           <string>Hall table 1 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hall2Box">
          <property name="toolTip">
           <string>Hall table 2 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hall3Box">
          <property name="toolTip">
           <string>Hall table 3 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hall4Box">
          <property name="toolTip">
           <string>Hall table 4 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hall5Box">
          <property name="toolTip">
           <string>Hall table 5 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hall6Box">
          <property name="toolTip">
           <string>Hall table 6 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hall7Box">
          <property name="toolTip">
           <string>Hall table 7 position (0 - 200, 255 = not used)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
